.github/workflows/genesis-test.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     steps:
       - uses: actions/setup-go@v2.1.3
         with:
-          go-version: 1.16
+          go-version: 1.17
```

Comment:
Upgrade go version to avoid build error
```sh
Error: ../../../go/pkg/mod/github.com/!cosm!wasm/wasmvm@v1.0.0-beta7/lib.go:7:2: //go:build comment without // +build comment
```
Fix this error by upgrading the go version
https://www.reddit.com/r/golang/comments/rn25ov/encountering_gobuild_comment_without_build/